api/install/install.go

```diff
@@ -46,6 +68,18 @@ type service interface {
 	Install(ctx context.Context, req Request) (Response, error)
 }
 
+// Handler handles an install request
+// swagger:route PUT /install installRelease
+//
+// Installs a helm release as specified in the request
+//
+// consumes:
+//	- application/json
+// produces:
+// 	- application/json
+// schemes: http
+// responses:
+//   200: installResponse
```

Comment:
Can we document non-ok responses?

Reply:
currently all our responses have the same struct so even documenting them does not help. We would need to create different responses for different http codes so that we could document them well. Since, that will require much more code change have skipped that for now. 
Nonetheless, will add all possible responses to the doc.